static/js/composebox_typeahead.js

```diff
@@ -485,7 +485,7 @@ exports.get_person_suggestions = function (query, opts) {
     const cutoff_length = exports.max_num_items;
 
     const filtered_message_persons = filter_persons(
-        people.get_message_people()
+        people.get_active_message_people()
     );
```

Comment:
Is it easy to include a test for the new behavior in the `composebox_typeahead` tests?  I think that'd be a valuable record of our intent for how this should work, and may help prevent regressions in future refactoring.

Reply:
@timabbott added the test. Let me know if the tests are fine or not.